db/migrate/20201006061413_drop_column_email_from_user.rb

```diff
@@ -0,0 +1,6 @@
+class DropColumnEmailFromUser < ActiveRecord::Migration[6.0]
```

Comment:
name ends with plural `Users` as we use table name, you can check other migrations.